discovery-provider/src/models/models.py

```diff
@@ -884,14 +884,15 @@ def __repr__(self):
 class ChallengeType(str, enum.Enum):
     boolean = 'boolean'
     numeric = 'numeric'
+    aggregate = 'aggregate'
+    trending = 'trending'
```

Comment:
does it makes sense for this to just a be "timeseries" type challenge or something like that?

Reply:
thought a while about this - I'm not sure whether trending should be generalized into a time series challenge, or just a "hidden if not complete" challenge, or what. Ultimately figured it made sense not to abstract this until we have another concrete example of how it would be used